backend/typescript/rest/camperRoutes.ts

```diff
@@ -76,4 +76,20 @@ camperRouter.get("/", async (req, res) => {
   }
 });
 
+camperRouter.get("/refund-confirm/:chargeId", async (req, res) => {
+  const { chargeId } = req.params;
+  try {
+    const camper = await camperService.getCamperByChargeId(
+      (chargeId as unknown) as number,
+    );
+    res.status(200).json(camper);
+  } catch (error: unknown) {
+    if (getErrorMessage(error).includes("Camper with Charge Id")) {
```

Comment:
this works but is a bit jank :'D would it be possible to use the cause property on the Error object before throwing it in the camper service and check that property here? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error

Reply:
for sure!

Reply:
returning just the one 500 error works tooo, we can leave it as it